lib/spack/docs/module_file_support.rst

```diff
@@ -500,20 +500,40 @@ version of python a set of python extensions is associated with. Likewise, the
 ``openblas`` string is attached to any program that has openblas in the spec,
 most likely via the ``+blas`` variant specification.
 
+The most heavyweight solution to module naming is to change the entire
+naming convention for module files. This uses the projections format
+covered in :ref:`adding_projections_to_views`.
+
+.. code-block:: yaml
+
+  modules:
+    tcl:
+      projections:
```

Comment:
Consistency question: for views it makes sense to me to call this a "projection", as you're specifying how a bunch of symlinks should look.  For modules should it just be `formats`?  I think the latter is more intuitive for new users (though I have no evidence).  Would it make sense to call both `formats` eventually, or is it better to just be consistent with views for now?

Reply:
I agree that "projection" doesn't apply as cleanly here, but I still think it's okay (and I wouldn't want to call it "formats" in the view context). I wanted to keep it the same as in the view context because I think that helps users understand what their doing when they swap from one to the other, but I'm not committed to that if you think it's too unclear what's going on.

Reply:
@adamjstewart: you usually have a good feel for these things.  What's intuitive?

Reply:
I like the idea of keeping things consistent with views, but I admittedly haven't used modules in a while so I may not be the best person to ask.

Reply:
Good enough